src/main/java/com/openshift/internal/restclient/http/UrlConnectionHttpClientBuilder.java

```diff
@@ -44,6 +48,12 @@ public UrlConnectionHttpClientBuilder setCredentials(String username, String pas
 		return setAuthorizationStrategy(new BasicAuthorizationStrategy(username, password, token));
 	}
 	
+	public UrlConnectionHttpClientBuilder setCertificate(String alias, X509Certificate cert) {
```

Comment:
Definitely based on the KeyStore API it is a alias/cert pair.

My experience isn't significant either, but so far the client side scenario has been a single cert.  But I'm not necessarily opposed to say a Map<String,X509Certificate> input parameter structure if you feel strongly enough about it.